packages/shared-metrics/src/util/nativeModuleRetry.js

```diff
@@ -183,23 +183,14 @@ function copyPrecompiled(opts, loaderEmitter, callback) {
       .then(() => {
         // See below for the reason why we append 'precompiled' to the path.
         const targetDir = path.join(opts.nativeModulePath, 'precompiled');
+        const sourceDir = path.join(os.tmpdir(), opts.nativeModuleName);
+        logger.debug(
+          `Copying the precompiled build for ${opts.nativeModuleName} ${label} from ${sourceDir} to ${targetDir}.`
+        );
 
-        // @ts-ignore
-        copy(
-          path.join(os.tmpdir(), opts.nativeModuleName),
-          targetDir,
-          {
-            overwrite: true,
-            dot: true
-          },
-          // @ts-ignore
-          cpErr => {
-            if (cpErr) {
-              logger.warn(`Copying the precompiled build for ${opts.nativeModuleName} ${label} failed.`, cpErr);
-              callback(false);
-              return;
-            }
-
+        fse
+          .copy(sourceDir, targetDir, { dereference: true })
```

Comment:
Q: Why do we need `dereference: true`?

> dereference (boolean): dereference symlinks, default is false.

Reply:
I encountered an error while running the tests in local:

> Error: Cannot copy '/usr/bin/python3' to a subdirectory of itself

Following the advice in the comment [here](https://github.com/chegele/AutoGitUpdate/issues/6#issuecomment-948901575), I have used this option.

I will verify by removing the option to see if the issue is specific to my local environment.

Reply:
Ok! Thx

Reply:
The reason to **not** use `dereference: true ` is that `recursive-copy` does [not](https://github.com/timkendrick/recursive-copy/blob/2.0.14/lib/copy.js#L280) dereference symlinks. It copies the symlink itself, not the file or directory it points to.

Reply:
Yes, setting `dereference: true` is not necessary in our case. The issue was specific to my local machine.

Reply:
Thanks for catching this ❤️

Reply:
MUA